src/frontends/paddle/src/op/elementwise_ops.cpp

```diff
@@ -46,6 +46,32 @@ NamedOutputs elementwise_greater_equal(const NodeContext& node_context) {
     return elementwise_ops<default_opset::GreaterEqual>(node_context);
 }
 
+NamedOutputs elementwise_floordiv(const NodeContext& node_context) {
+    auto x = node_context.get_input("X");
+    auto y = node_context.get_input("Y");
+
+    const auto axis = node_context.get_attribute<int>("axis", -1);
+
+    PADDLE_OP_CHECK(node_context, x.get_partial_shape().rank().is_static(), "elementwise_ops: X rank must be static!");
+    PADDLE_OP_CHECK(node_context, y.get_partial_shape().rank().is_static(), "elementwise_ops: Y rank must be static!");
+    int64_t x_rank = x.get_partial_shape().rank().get_length();
+    int64_t y_rank = y.get_partial_shape().rank().get_length();
+
+    if ((axis == -1) || (axis == x_rank - 1) || (x_rank == y_rank)) {
+        return node_context.default_single_output_mapping({std::make_shared<default_opset::Floor>(std::make_shared<default_opset::Divide>(x, y))}, {"Out"});
+    } else {
+        std::vector<int64_t> indices;
+        for (int64_t i = 0; i < axis; i++)
+            indices.push_back(i);
+        for (int64_t i = y_rank + axis; i < x_rank; i++)
+            indices.push_back(i);
+
+        auto indices_node = default_opset::Constant::create(ov::element::i64, ov::Shape{indices.size()}, indices);
+        auto y_node = std::make_shared<default_opset::Unsqueeze>(y, indices_node);
+        return node_context.default_single_output_mapping({std::make_shared<default_opset::Floor>(std::make_shared<default_opset::Divide>(x, y_node))}, {"Out"});
```

Comment:
Although the https://www.paddlepaddle.org.cn/documentation/docs/zh/1.8/api_cn/layers_cn/elementwise_floordiv_cn.html#canshu show that the input data type support 'float32' and 'float64', but I check the code in Paddle, https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/elementwise/elementwise_floordiv_op.cc , it only support int32 and in64. If it only supports 'int32' and 'int64', maybe we can use the default_opset::Divide with 'pythondiv' = True instead of using std::make_shared<default_opset::Floor>(std::make_shared<default_opset::Divide>.

Reply:
已修改默认使用Divide